modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -10,42 +10,35 @@
 
 [NOTE]
 ====
-Workload partitioning can only be enabled during cluster installation. You cannot disable workload partitioning post-installation. However, you can reconfigure workload partitioning by updating the `cpu` value that you define in the performance profile, and in the related `MachineConfig` custom resource (CR).
+Workload partitioning can be enabled during cluster installation only.
+You cannot disable workload partitioning post-installation.
```

Comment:
however the set of CPUs assigned to the isolated and reserved sets can be modified through the PerformanceProfile. Changes to the cpu settings will result in a reboot of the node(s).